cmake/CompilerWarnings.cmake

```diff
@@ -3,7 +3,7 @@
 # https://github.com/lefticus/cppbestpractices/blob/master/02-Use_the_Tools_Available.md
 
 # Helper function to enable compiler warnings for a specific set of files
-function(set_file_warnings)
+function(set_file_warnings target)
```

Comment:
The description in comment and function name were not updated.

Reply:
Good point. The function name is out of date too. I fixed the comment and function name.